docs/user-guide/admin-cli.md

```diff
@@ -39,17 +39,17 @@ optional arguments:
 ### Create
 A tenant named "_DEFAULT" will be created automatically post install.
 
-Creates a new named tenant and optionally assigns VMs.
+Creates a new named tenant and optionally assigns VMs. Valid tenant name is only allowed to be "[a-zA-Z0-9_][a-zA-Z0-9_.-]*"
```

Comment:
@lipingxue  a quick question: PR description says ```Fixes issue 816 (space in DS names).``` can you please help to understand why some of the fixes are related with ```TENANT_NAME_INVALID``` or ```VOLUME_NAME_INVALID```?